tests/api_e2e/analytics.test.js

```diff
@@ -1,13 +1,14 @@
-const e = require('express')
 const request = require('supertest')
-const baseURL = 'http://localhost:8080'
-
+const fs = require('fs')
 const auth = require('./authheader')
 
+const baseURL = 'http://localhost:8080'
+const path_prefix = '/analytics/v1/3/'
```

Comment:
Should we move the user id into a separate (environment) variable? What if we want to change it in the future? We might forget to change it here.

Reply:
done